import unittest2


class TestCodeReview(unittest2.TestCase):
    def test_review(self):
        # self.fail("Guaranteed Failure")
        pass
